source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,116 @@
+
+It's a platform (OS) agnostic CLI tool and supports multiple hypervisors like VirtualBox, Xhyve (Mac OS), VMWare Fusion (Mac OS) and HyperV (Windows). Detailed instructions about it can be found in the [Minishift documentation](https://github.com/Minishift/minishift#prerequisites).
+
+## Installing Minishift
+
+1. Download the archive for your OS from the Minishift [releases page](https://github.com/minishift/minishift/releases) and unpack it.
+1. Copy the contents of the directory to your preferred location.
+1. Add the minishift binary to your PATH environment variable.
+
+To install in specific OS, refer to the [minishift installation guide](https://github.com/Minishift/minishift#installing-minishift).
+
+## Quickstart with Minishift
+
+1. Get start as follows:
+
+    ```
+    $ minishift start
+     Starting local OpenShift cluster using 'kvm' hypervisor...
+     ...
+        OpenShift server started.
+        The server is accessible via web console at:
+            https://192.168.99.128:8443
+
+        You are logged in as:
+            User:     developer
+            Password: developer
+
+        To login as administrator:
+            oc login -u system:admin
+    ```
+
+    This will:
+    - Download the latest [_minishift-b2d-iso_](https://github.com/Minishift/minishift-b2d-iso)
+    - Starts a VM using libmachine
+    - Download OpenShift client binary (`oc`) with version `1.4.1`
+    - Cache both `oc` and ISO into your `$HOME/.minishift/cache` folder
+    - Finally provisions OpenShift single node cluster
+
+    **Note:** This step also logs you in as `developer` in the `myproject` project.
+
+1. Export the OpenShift client binary (`oc`) to the _PATH_ environment variable as follows:
+
+    ```
+    $ export PATH=$PATH:$HOME/.minishift/cache/oc/v1.4.1
+    ```
+
+1. Deploy a sample application as follows:
+    - Create a Nodejs example application:
+
+         ```
+         $ oc new-app https://github.com/openshift/nodejs-ex -l name=myapp
+         ```
+    - Track the build log until the app is built and deployed using:
+
+        ```
+        $ oc logs -f bc/nodejs-ex
+        ```
+    - Expose a route to the service as follows:
+
+        ```
+        $ oc expose svc/nodejs-ex
+        ```
+    - Open the application in a browser as follows:
+
+        ```
+        $ minishift openshift service nodejs-ex -n myproject
+        ```
+
+1. When done with building the application for the day, stop the cluster as follows:
+
+    ```
+     $ minishift stop
+     Stopping local OpenShift cluster...
+     Stopping "minishift"...
+    ```
+
+    If you want to access the same cluster some other day, use `minishift start`.
+
+1. At anytime if you want to delete the cluster, use the following command:
+
+    ```
+    $ minishift delete
+    ```
+
+Refer [Minishift's README](https://github.com/Minishift/minishift#minishift) to get more detailed description of getting start with single node OpenShift cluster.
+
```

Comment:
"Refer to the [Minishift README]" (also, pinning this link TBC when we publish docs)